avl/tree.go

```diff
@@ -376,14 +378,27 @@ func (t *Tree) iterateDiff(prevViewID uint64, callback func(n *node) bool) {
 	}
 }
 
-func (t *Tree) DumpDiff(prevViewID uint64) []byte {
+// DumpDiffAll is a wrapper of DumpDiff which simply
+// dumps the diff into memory (similar to ioutil.ReadAll).
+func (t *Tree) DumpDiffAll(prevViewID uint64) []byte {
```

Comment:
maybe `DumpDiffFull` or `DumpWholeDiff` or something similar
i got the reference to `ReadAll` but DiffAll just doesn't make much sense